ci/build-docker-image.sh

```diff
@@ -33,23 +63,30 @@ case $IMAGE_NAME_SUBSTRING in
 esac
 
 # Create tag based on branch name
-IMAGE_NAME=tuerobotics/"$IMAGE_NAME_SUBSTRING":$(echo "$TRAVIS_BRANCH" | tr '[:upper:]' '[:lower:]' | sed -e 's:/:_:g')
-
-echo -e "\e[35m\e[1m Creating docker $IMAGE_NAME \e[0m"
+CI_DOCKER_IMAGE_NAME="$CI_DOCKER_IMAGE_NAME":$(echo "$CI_BRANCH" | tr '[:upper:]' '[:lower:]' | sed -e 's:/:_:g')
+echo -e "\e[35m\e[1m Creating docker $CI_DOCKER_IMAGE_NAME \e[0m"
 
 # build the Docker image (this will use the Dockerfile in the root of the repo)
-docker build --build-arg BRANCH="${TRAVIS_PULL_REQUEST_BRANCH:-$TRAVIS_BRANCH}" --build-arg \
-PULL_REQUEST="$TRAVIS_PULL_REQUEST" --build-arg COMMIT="$TRAVIS_COMMIT" --build-arg \
-CI="$CI" --build-arg BASE_IMAGE="$BASE_IMAGE" -t "$IMAGE_NAME" .
+if [ "$CI_DOCKER_SSH" == "true" ]
+then
+    DOCKER_BUILDKIT=1 docker build --ssh=default --build-arg BRANCH="${CI_PULL_REQUEST_BRANCH:-$CI_BRANCH}" --build-arg \
+        PULL_REQUEST="$CI_PULL_REQUEST" --build-arg COMMIT="$CI_COMMIT" --build-arg \
+        CI="$CI" --build-arg BASE_IMAGE="$BASE_IMAGE" -t "$CI_DOCKER_IMAGE_NAME" .
+else
+    docker build --build-arg BRANCH="${CI_PULL_REQUEST_BRANCH:-$CI_BRANCH}" --build-arg \
+        PULL_REQUEST="$CI_PULL_REQUEST" --build-arg COMMIT="$CI_COMMIT" --build-arg \
+        CI="$CI" --build-arg BASE_IMAGE="$BASE_IMAGE" -t "$CI_DOCKER_IMAGE_NAME" .
+fi
```

Comment:
Problem in using `DOCKER_BUILDKIT=1` when not using ssh?

Reply:
Fixed in new commits